tests/README.md

```diff
@@ -209,36 +204,45 @@ allure serve ./reports
 ### Integration Mode
 
 1. Build tests docker image and upload to kubernetes cluster
-```shell
-just build-test-docker
-just load-tests-image-kind
-```
+
+    ```shell
+    just build-test-docker
+    just load-tests-image-kind
+    ```
+
 2. If **SUT** is not deployed initiate:
-```shell
-just deploy-tests-helm-ci pre_merge
-```
+
+    ```shell
+    just deploy-tests-helm-ci pre_merge
+    ```
+
 3. Execute tests
-```shell
-just run-tests
-```
+
+    ```shell
+    just run-tests
+    ```
+
 4. Investigate reports
-```shell
-allure serve <path to raw result data>
-```
+
+    ```shell
+    allure serve <path to raw result data>
+    ```
 
 ## CI-CD Workflows
 
 Current usage of test project is in the following ci flows:
+
 - cloudbeat-ci
-  - build cloudbeat-tests docker image
-  - load cloudbeat-tests docker image to kind cluster
-  - deploy tests helm chart
-  - execute product and integration cloudbeat tests
+    - build cloudbeat-tests docker image
+    - load cloudbeat-tests docker image to kind cluster
+    - deploy tests helm chart
+    - execute product and integration cloudbeat tests
```

Comment:
i _think_ this could be a comment in the command itself, removing the need for this section